src/fetch/README.md

```diff
@@ -165,3 +165,13 @@ it('mocks the fetch', async () => {
 ```
 
 Why mocking `global.fetch` doesn't work? It's because this fetch doesn't use or pollute `global` variable: it uses ponyfill instead of polyfill behind the scenes.
+
+
+## Msw, an alternative to mock fetch
```

Comment:
Is this a good place where to put it? Under "Frequently Asked Questions"? 🤔 It's not a question. Maybe `###` as a subsection to the answer above?

Reply:
Yeah, you are right, subsection is better here, it belongs under the mock section.